### ISSUE TITLE
Helm doesn't uninstall open telemetry collector 

### ISSUE THREAD

**Issue by mathisve** [timescale/tobs#601]
**What happened?**

I'm not sure if this is a bug or if this is intentional, but when you deploy tobs using helm with the opentelemetry-operator enabled, it doesn't delete the opentelemetry-operator and pods associated with the otel operator.

**Did you expect to see something different?**

I expect to see the otel related resources (minus CRDs) to be deleted when uninstalling tobs using helm. 

**How to reproduce it (as minimally and precisely as possible)**:

```
helm install tobs timescale/tobs -n tobs --set opentelemtry-operator.enabled=true

helm ls -n tobs
# NAME    NAMESPACE       REVISION        UPDATED                                 STATUS          CHART           APP VERSION
# tobs    tobs            1               2022-10-06 13:29:27.823741 -0600 MDT    deployed        tobs-14.13.1               

helm uninstall tobs -n tobs

kubectl get pods -n tobs
# NAME                                            READY   STATUS    RESTARTS   AGE
# tobs-opentelemetry-collector-74bcfbcfcd-d5rmq   1/1     Running   0          3m3s
```

**Environment**

* tobs version:

  `tobs-14.13.1 (helm chart)`

    

    

* Kubernetes version information:

  `Client Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.3", GitCommit:"aef86a93758dc3cb2c658dd9657ab4ad4afc21cb", GitTreeState:"clean", BuildDate:"2022-07-13T14:21:56Z", GoVersion:"go1.18.4", Compiler:"gc", Platform:"darwin/arm64"}
Kustomize Version: v4.5.4
Server Version: version.Info{Major:"1", Minor:"22+", GitVersion:"v1.22.11-eks-18ef993", GitCommit:"b9628d6d3867ffd84c704af0befd31c7451cdc37", GitTreeState:"clean", BuildDate:"2022-07-06T18:06:23Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}`

    

* Kubernetes cluster kind:

  `eksctl`

**Comment by nhudson** [comment]
@mathisve this is an issue with Helm from what we can tell.  The issue got closed somehow, but it's described here #365 specifically this comment https://github.com/timescale/tobs/issues/365#issuecomment-1171152080

The docs indicate specific steps you need to follow to remove everything from the namespace https://github.com/nhudson/tobs/blob/main/chart/README.md#uninstall

**Comment by alejandrodnm** [comment]
Hi @mathisve I'm closing this. Feel free to reopen it if the need arise. Cheers.
